gm4_audere_shamir/data/gm4_audere_shamir/functions/store_maximum.mcfunction

```diff
@@ -0,0 +1,34 @@
+execute if entity @s[nbt={Item:{id:"minecraft:wooden_hoe"}}] run data modify entity @s Item.tag.MaxDurability set value 59
+execute if entity @s[nbt={Item:{id:"minecraft:wooden_axe"}}] run data modify entity @s Item.tag.MaxDurability set value 59
+
+execute if entity @s[nbt={Item:{id:"minecraft:stone_shovel"}}] run data modify entity @s Item.tag.MaxDurability set value 131
+execute if entity @s[nbt={Item:{id:"minecraft:stone_pickaxe"}}] run data modify entity @s Item.tag.MaxDurability set value 131
+execute if entity @s[nbt={Item:{id:"minecraft:stone_hoe"}}] run data modify entity @s Item.tag.MaxDurability set value 131
+execute if entity @s[nbt={Item:{id:"minecraft:stone_axe"}}] run data modify entity @s Item.tag.MaxDurability set value 131
+
+execute if entity @s[nbt={Item:{id:"minecraft:iron_shovel"}}] run data modify entity @s Item.tag.MaxDurability set value 250
+execute if entity @s[nbt={Item:{id:"minecraft:iron_pickaxe"}}] run data modify entity @s Item.tag.MaxDurability set value 250
+execute if entity @s[nbt={Item:{id:"minecraft:iron_hoe"}}] run data modify entity @s Item.tag.MaxDurability set value 250
+execute if entity @s[nbt={Item:{id:"minecraft:iron_axe"}}] run data modify entity @s Item.tag.MaxDurability set value 250
+
+execute if entity @s[nbt={Item:{id:"minecraft:diamond_shovel"}}] run data modify entity @s Item.tag.MaxDurability set value 1561
+execute if entity @s[nbt={Item:{id:"minecraft:diamond_pickaxe"}}] run data modify entity @s Item.tag.MaxDurability set value 1561
+execute if entity @s[nbt={Item:{id:"minecraft:diamond_hoe"}}] run data modify entity @s Item.tag.MaxDurability set value 1561
+execute if entity @s[nbt={Item:{id:"minecraft:diamond_axe"}}] run data modify entity @s Item.tag.MaxDurability set value 1561
+
+execute if entity @s[nbt={Item:{id:"minecraft:netherite_shovel"}}] run data modify entity @s Item.tag.MaxDurability set value 2031
+execute if entity @s[nbt={Item:{id:"minecraft:netherite_pickaxe"}}] run data modify entity @s Item.tag.MaxDurability set value 2031
+execute if entity @s[nbt={Item:{id:"minecraft:netherite_hoe"}}] run data modify entity @s Item.tag.MaxDurability set value 2031
+execute if entity @s[nbt={Item:{id:"minecraft:netherite_axe"}}] run data modify entity @s Item.tag.MaxDurability set value 2031
+
+execute if entity @s[nbt={Item:{id:"minecraft:golden_shovel"}}] run data modify entity @s Item.tag.MaxDurability set value 33
+execute if entity @s[nbt={Item:{id:"minecraft:golden_pickaxe"}}] run data modify entity @s Item.tag.MaxDurability set value 33
+execute if entity @s[nbt={Item:{id:"minecraft:golden_hoe"}}] run data modify entity @s Item.tag.MaxDurability set value 33
+execute if entity @s[nbt={Item:{id:"minecraft:golden_axe"}}] run data modify entity @s Item.tag.MaxDurability set value 33
+
+execute if entity @s[nbt={Item:{id:"minecraft:shears"}}] run data modify entity @s Item.tag.MaxDurability set value 237
```

Comment:
It would also be advantageous to implement a success score based if-else here. That would avoid unnecessary nbt checks once one of the checks has succeeded.